src/organizations/organization.service.ts

```diff
@@ -145,22 +146,31 @@ export class OrganizationService {
     invitee: string,
     answer: boolean,
   ) {
+
+    if (answer === undefined) {
```

Comment:
given the typescript validation, here it should never be undefined, but a boolean. if it is undefined, it means that the validation on controller let it slip.